integration-test/src/main/java/com/scalar/db/api/DistributedStorageAdminImportTableIntegrationTestBase.java

```diff
@@ -222,6 +226,43 @@ public void alterColumnType_WideningConversion_ForImportedTable_ShouldAlterPrope
     }
   }
 
+  @Test
+  public void dropNamespace_ShouldNotDropNonScalarDBTables() throws Exception {
```

Comment:
Since we've added a similar test in `DistributedStorageAdminIntegrationTestBase`, do we still need this test?